themes/jquery/footer.php

```diff
@@ -36,7 +36,12 @@
 	</div>
 </footer>
 
-<?php wp_footer(); ?>
+<?php
+	wp_footer();
+
+	if ( function_exists( 'footer_append' ) )
+		footer_append();
```

Comment:
I'm not sure what problem you were having enqueueing these files, but replacing the function definitions in `page.php` with the following code works just fine for me:

```php
wp_enqueue_style('jquery-ui', 'https://code.jquery.com/ui/1.11.4/themes/ui-lightness/jquery-ui.css');
wp_enqueue_style('sri-modal', get_template_directory_uri() . '/css/sri-modal.css');
wp_enqueue_script('jquery-ui', 'https://code.jquery.com/ui/1.11.4/jquery-ui.min.js');
wp_enqueue_script('clipboard', get_template_directory_uri() . '/js/clipboard.1.5.5.min.js');
wp_enqueue_script('sri-modal', get_template_directory_uri() . '/js/sri-modal.js');
```